plugins/scaffolder/api-report.md

```diff
@@ -376,6 +388,9 @@ export type TaskPageProps = {
   loadingText?: string;
 };
 
+// @public
+export type TasksOwnerFilterKind = 'owned' | 'all';
```

Comment:
Wonder if we should skip this extra type for now? and just inline the union.

Feeling the naming is a little bit off, was expecting something like `'me' | 'anyone'` or something like that?

Little bit scary that this conflicts with an entity ref string in case we want to be able to query for other users in the future too. Perhaps we could switch the naming of the key here to leave `createdBy` available? Thinking something like `filterByOwnership`, and in that case `'owned' | 'all'` probably makes more sense too. Then again that's all of a sudden talking about "ownership", while it's otherwise only from the point of view of who created the task.